File: beecrowd/c++/iniciante/ex1018.cpp
#include <iostream>
using namespace std;

int main(){
    int cedulas[7] = {100, 50, 20, 10, 5, 2, 1};
    int n, qtd;
    cin>>n;
    cout<<n<<endl;
    for (int i=0; i<7; i++){
        qtd = n / cedulas[i];
        n = n % cedulas[i];
        cout<<qtd<<" nota(s) de R$ "<<cedulas[i]<<",00"<<endl;
    }

    return 0;
}

File: beecrowd/python/iniciante/ex1018.py
cedulas = [100, 50, 20, 10, 5, 2, 1]
n = int(input())
print(n)
for cedula in cedulas:
    qtd = n // cedula
    n = n % cedula
    print(f"{qtd} nota(s) de R$ {cedula},00", end='\n')